views/movies/movies.hbs

```diff
@@ -0,0 +1,18 @@
+<h2>Movies</h2>
+
+<form action="/movies/create" method="post">
```

Comment:
The `movies.hbs` should be a page with all the movies, what you did here should be in the `new-movie.hbs`